app/controllers/order_management/order_details_controller.rb

```diff
@@ -101,7 +101,7 @@ def load_order_statuses
     return if @order_detail.reconciled?
 
     if @order_detail.complete?
-      @order_statuses = [OrderStatus.complete, OrderStatus.canceled]
+      @order_statuses = [OrderStatus.complete, OrderStatus.canceled, OrderStatus.unrecoverable]
```

Comment:
This line is making 3 db queries and possibly one more in the following, maybe we can refactor this to make just one. I know this is not actually related to this ticket so it's not blocking at all.